homeassistant/components/binary_sensor/__init__.py

```diff
@@ -35,6 +35,7 @@
     'smoke',         # Smoke detector
     'sound',         # On means sound detected, Off means no sound
     'vibration',     # On means vibration detected, Off means no vibration
+    'plug',          # On means plugged in, Off means unplugged
```

Comment:
Please put this line sorted :abcd: in the list.

Reply:
Thanks, changed.